tests/test_cli.py

```diff
@@ -41,13 +37,18 @@ def test_init_multiple_output_matrix(testing_workdir):
     parser = argparse.ArgumentParser()
     subparser = parser.add_subparsers()
     init_obj = cli.Init(subparser)
+    regen_obj = cli.Regenerate(subparser)
     recipe = os.path.join(_thisdir, 'recipes', 'multiple_outputs')
     feedstock_dir = os.path.join(testing_workdir, 'multiple-outputs-test-feedstock')
     args = InitArgs(recipe_directory=recipe,
-                    feedstock_directory=feedstock_dir,
-                    no_git_repo=False,
-                    variant_config_files=os.path.join(recipe, 'conda_build_config.yaml'))
+                    feedstock_directory=feedstock_dir)
     init_obj(args)
+    with open(os.path.join(feedstock_dir, "conda-forge.yml"), "w") as f:
+        f.write("exclusive_config_file: recipe/conda_build_config.yaml")
```

Comment:
Sorry not following this part. Why are we using the `recipe`’s `conda_build_config.yaml` as the `exclusive_config_file`?

Reply:
why avoid it?  The one in the recipe will override it anyway.

Reply:
current test assumes that only the one in the recipe is used and there are no 2 different platforms for windows (win-32 and win-64). so, to keep the test as it is, I'm avoiding  conda-forge-pinning's conda_build_config.yaml

Reply:
OK, that's fine.  This is only a test.  It's OK to do squirrelly things.  I definitely do not recommend doing this generally.  Should we put a comment here to clarify that to anyone who reads this code?

Reply:
Yeah ideally I'd we should move this file a directory out or similar, but at the end of the day this is just the tests. A comment would certainly be nice so that people don't get the wrong impression of how this option should be used.